app/src/main/java/app/nexd/android/ui/helper/transcript/articles/TranscriptArticlesFragment.kt

```diff
@@ -7,18 +7,18 @@ import android.view.ViewGroup
 import androidx.fragment.app.Fragment
 import androidx.lifecycle.Observer
 import androidx.navigation.fragment.findNavController
-import androidx.navigation.navGraphViewModels
 import androidx.recyclerview.widget.LinearLayoutManager
 import app.nexd.android.R
 import app.nexd.android.databinding.FragmentTranscriptArticlesBinding
+import app.nexd.android.di.sharedGraphViewModel
 import app.nexd.android.ui.helper.transcript.TranscriptViewModel
 import kotlinx.android.synthetic.main.fragment_transcript_articles.*
 import mva2.adapter.ListSection
 import mva2.adapter.MultiViewAdapter
 
 class TranscriptArticlesFragment : Fragment() {
 
-    private val viewModel: TranscriptViewModel by navGraphViewModels(R.id.nav_transcript)
+    private val viewModel: TranscriptViewModel by sharedGraphViewModel(R.id.nav_transcript)
```

Comment:
in case we need a single fragment scoped, non-shared view model, maybe for data binding on a single screen, I would name this already as something that is shared across several fragments in the graph: `transcriptViewModel`.

Reply:
Yep, good point. I'll rename it

Reply:
done